File: aliyun-java-best-practise/src/main/java/com/aliyun/bestpractise/StartInstanceDemo.java
package com.aliyun.bestpractise;

import com.aliyuncs.DefaultAcsClient;
import com.aliyuncs.IAcsClient;
import com.aliyuncs.ecs.model.v20140526.StartInstanceRequest;
import com.aliyuncs.ecs.model.v20140526.StartInstanceResponse;
import com.aliyuncs.exceptions.ClientException;
import com.aliyuncs.profile.DefaultProfile;

/**
 * 控制ECS服务器的启动
 */
public class StartInstanceDemo {
    public static void main(String[] args) {
        DefaultProfile profile = DefaultProfile.getProfile(
                "RegionId",
                "accessKeyId",
                "accessKeySecret");
        IAcsClient client = new DefaultAcsClient(profile);
        StartInstanceRequest request = new StartInstanceRequest();
        request.setInstanceId("ECS实例id");
        StartInstanceResponse response;
        try {
            response =  client.getAcsResponse(request);
        } catch (ClientException e) {
            e.printStackTrace();
        }
    }
}

File: aliyun-python-best-practise/控制ECS服务器的启动和停止/startInstance_demo.py
from aliyunsdkcore.client import AcsClient
from aliyunsdkecs.request.v20140526.StartInstanceRequest import StartInstanceRequest
client = AcsClient('access_key_id', 'access_key_secret', 'region_id')
request = StartInstanceRequest()
request.set_InstanceId('ECS实例ID')
response = client.do_action_with_exception(request)
print(response)
